fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -236,7 +235,7 @@ public void destroy() {
     // TODO do not use transactions for internal reads
     private Transaction transaction() {
         if (transaction == null) {
-            transaction = sessionFactory.getNewTransaction();
+            transaction = txManager.create();
```

Comment:
Also don't think this should mint a new transaction, it should try to reuse the ongoing transaction from the request

Reply:
I will refactor this PR to no use transaction once #1561 is merged.

Reply:
Updated to existing transaction associated with the request.